### DISCUSSION TITLE
Can/should this replace gitter?

### DISCUSSION THREAD

**Discussion by poire-z** [koreader/koreader#6966] [General]
Just asking to see how this work :)

**Comment by Frenzie** [comment]
mayyybe, looks more like GH comments without an attached issue

**Comment by NiLuJe** [comment]
i.e., a forum? :D.

**Reply by Frenzie** [reply]
Yeah, pretty much.

**Comment by poire-z** [comment]
Well, not really interesting then.
They don't appear when you search thru the issue/PR searchbox ("mayyybe" :), they don't appear in the api results (which returns both issues and PRs)... and the messages look too spread. So best to keep opening issues...

**Comment by pazos** [comment]
> Just asking to see how this work :)

Not sure if it can replace gitter, but I like how homebrew use discussions to redirect people with questions (and how they add statements in bug reports too). See https://github.com/Homebrew/brew/issues/new/choose

Question as issues don't make much sense to me. They get answered and closed. Then somebody ask them again ad infinitum.
